course-v2/layouts/partials/mobile_course_nav.html

```diff
@@ -1,6 +1,25 @@
 <div
   id="mobile-course-nav"
-  class="navbar-offcanvas medium-and-below-only drawer">
-  <div class="mb-4 mt-4">Browse Course Material</div>
+  class="navbar-offcanvas offcanvas-toggle offcanvas-toggle-close medium-and-below-only drawer">
+  <div class="mb-4 mt-4">
+    <div class="d-flex align-items-center">
+      <div>Browse Course Material</div>
```

Comment:
IMO a header[^1] would be better here. I took a quick stab and also got rid of some of the divs. (and swapped mb+mt for my):
```html
    <h3 class="my-4 d-flex align-items-center justify-content-between">
      Browse Course Material
      <button
        class="btn"
        type="button"
        aria-label="Close Course Menu"
        onclick="$('#mobile-course-nav-toggle').click();"
        >
        <img
          src="/images/close_small.svg"
          alt=""
          width="12px"
        />
      </button>
    </h3>
```

Up to you, but I would get rid of the unnecessary class.

Reply:
Thanks @ChristopherChudzicki 
I have made these changes and gonna merge this PR now.